### ISSUE TITLE
Security warning when importing unsigned TokenScript definitions

### ISSUE THREAD

**Issue by micwallace** [AlphaWallet/alpha-wallet-android#2431]
When importing unsigned TokenScript files, at least on Android, there is no confirmation step. It wouldn't be hard to imagine this being used in a phishing scam. 

I recommend 1-2 things to mitigate this risk:

1. Implement a warning dialog when importing unsigned/unverified TokenScript definitions into AlphaWallet.
2. Restrict manual import to devices with developer mode enabled. I know this is at least possible in Android. This would be in addition to #1 to add an extra layer of security.

**Comment by JamesSmartCell** [comment]
Yes. In theory importing TokenScript files via clicking on them is only done during development of the scripts,

But yes we should restrict that, maybe in the TokenScript Management page you are required to allow debug scripts, and there should be a warning if a debug script attempts to send a transaction (note TokenScript can't ask for user to do legacy/pure signing, so in theory this should be safe - except possibly for a very targeted phishing scam where the user is known to operate on a specific multisig wallet, and somehow one of the other signatory's wallets has been compromised, and some other situation where doing a sign personal could cause transaction loss).

I will:
1. Add a debug mode to the TS management page which is popped open when a user first tries to load a debug script.
2. If a debug script attempts to send a transaction warn the user it may be unsafe.
